.travis.yml

```diff
@@ -1,37 +1,30 @@
 language: python
 sudo: false
 python:
-  - 2.7
-  - 3.4
   - 3.5
 cache:
   directories:
     - $HOME/.cache/pip
-env:
-  - DJANGO=django==1.8.14
-  - DJANGO=django==1.8.14 DB_ENGINE="django.db.backends.postgresql_psycopg2" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.8.14 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
-  - DJANGO=django==1.9.9
-  - DJANGO=django==1.9.9 DB_ENGINE="django.db.backends.postgresql" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.9.9 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
-  - DJANGO=django==1.10
-  - DJANGO=django==1.10 DB_ENGINE="django.db.backends.postgresql" DB_NAME="test_project" DB_USER="postgres"
-  - DJANGO=django==1.10 DB_ENGINE="django.db.backends.mysql" DB_NAME="test_project" DB_USER="travis"
 matrix:
   fast_finish: true
+env:
+  matrix:
+    - WORKER=python
+    - WORKER=flake8
```

Comment:
I'd make things nice and simple. Just run flake 8 as another tox env. There's very little to be gained running things in parallel.